### ISSUE TITLE
Lumina-mGPT support

### ISSUE THREAD

**Issue by Amazon90** [ollama/ollama#6254]
[Lumina-mGPT](https://github.com/Alpha-VLLM/Lumina-mGPT)

**Comment by igorschlum** [comment]
@Amazon90 I don't think that Ollama supports image generation LLM because the output is only text unless Ollama outputs base64 encoded images.

**Comment by Amazon90** [comment]
What software can support this model?

**Comment by igorschlum** [comment]
It's with Python for now.

https://github.com/Alpha-VLLM/Lumina-mGPT/blob/main/INSTALL.md

Not easy on a mac.
